posts/2021-02-11-Rust-1.50.0.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: "Announcing Rust 1.50.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.50.0. Rust is a
+programming language that is empowering everyone to build reliable and
+efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.50.0 is as easy as:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install]
+from the appropriate page on our website, and check out the
+[detailed release notes for 1.50.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1500-2021-02-11
+
+## What's in 1.50.0 stable
+
+For this release, we have improved array indexing, expanded union field safety, refined file descriptors, and added to the standard library.
```

Comment:
```suggestion
For this release, we have improved array indexing, expanded safe access to union fields, and added to the standard library.
```

* Not sure why, but "expanded union field safety" was not quite making sense to me - maybe this looks better?
* I don't thinking refining file descriptors parses for me, seems easier to cut it.